client/ayon_kitsu/hooks/pre_status_cahnge.py

```diff
@@ -0,0 +1,31 @@
+from ayon_applications import PreLaunchHook
+import os
+
+class pre_status_change(PreLaunchHook):
```

Comment:
```suggestion
class PreStatusChange(PreLaunchHook):
```
could we stick with the code style ? 
we use **PascalCase** also known as **UpperCamelCase**